lib/appsignal/config.rb

```diff
@@ -127,6 +129,10 @@ def write_to_environment
       ENV['APPSIGNAL_ENABLE_MINUTELY_PROBES']       = config_hash[:enable_minutely_probes].to_s
       ENV['APPSIGNAL_HOSTNAME']                     = config_hash[:hostname].to_s
       ENV['APPSIGNAL_PROCESS_NAME']                 = $0
+
+      unless config_hash[:ca_file_path] == DEFAULT_CONFIG[:ca_file_path]
```

Comment:
When a file does not exist or is not readable the Ruby process would crash. Now I don't see us using the Transmitter a lot (the only place this is used in Ruby), but i added a check for existence and readability, and if this not correct it ignores the setting and logs a warning.

would that work?
I didn't want to do this check when the config is loaded, because we do want to know what setting a user is using.

How we catch this on the agent side is another thing, depends on the http library implementation